src/gimelstudio/core/project_file.py

```diff
@@ -86,52 +86,56 @@ def SaveNodesData(self, nodes):
         self.contents["node_graph"]["nodes"] = nodes_data
 
     def CreateNodesFromData(self, nodegraph):
-        # Clear previous nodes and wires
-        nodegraph.nodes = {}
-        nodegraph.wires = []
+        try:
+            # Clear previous nodes and wires
+            nodegraph.nodes = {}
+            nodegraph.wires = []
 
-        # Create the nodes from the data
-        nodes_data = self.contents["node_graph"]["nodes"]
-        for node_id in nodes_data:
-            node_data = nodes_data[node_id]
+            # Create the nodes from the data
+            nodes_data = self.contents["node_graph"]["nodes"]
+            for node_id in nodes_data:
+                node_data = nodes_data[node_id]
 
-            # Create the node object
-            node = nodegraph.AddNode(node_data["idname"], 
+                # Create the node object
+                node = nodegraph.AddNode(node_data["idname"], 
                                     pos=node_data["pos"], 
                                     nodeid=node_id,
                                     location="POSITION")
 
-            # Create the node properties
-            prop_data = node_data["properties"]
-            for prop in prop_data:
-                node.NodeEditProp(idname=prop_data[prop]["idname"],
+                # Create the node properties
+                prop_data = node_data["properties"]
+                for prop in prop_data:
+                    node.NodeEditProp(idname=prop_data[prop]["idname"],
                                   value=prop_data[prop]["value"], 
                                   render=False)
```

Comment:
Code style: indent these lines
```suggestion
                    node.NodeEditProp(idname=prop_data[prop]["idname"],
                                      value=prop_data[prop]["value"], 
                                      render=False)
```